teradatalabs/mapr52-base/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN yum update -y \
 # INSTALL MapR 
   && yum install -y mapr-fileserver mapr-nfs mapr-nodemanager  mapr-cldb \ 
   && yum install -y mapr-zookeeper mapr-resourcemanager mapr-historyserver \  
-  && yum install -y mapr-webserver  mapr-gateway \
+  && yum install -y mapr-webserver  mapr-gateway mapr-httpfs \
```

Comment:
I am removing this commit since @AS186170 has already included it in mapr-52 base.